tests/setup.mjs

```diff
@@ -0,0 +1,10 @@
+import { register } from 'node:module';
+import { afterEach } from 'node:test';
+import { cleanup } from '@testing-library/react';
+
+register('./loader.mjs', import.meta.url);
+
+afterEach(cleanup);
+
+global.Event = window.Event;
+global.CustomEvent = window.CustomEvent;
```

Comment:
```suggestion
import { register } from 'node:module';
import { afterEach } from 'node:test';
import { GlobalRegistrator } from '@happy-dom/global-registrator';
import { cleanup } from '@testing-library/react';

register('@nodejs-loaders/alias', import.meta.url);
register('@nodejs-loaders/tsx', import.meta.url);
register('@nodejs-loaders/css-module', import.meta.url);

afterEach(cleanup);

GlobalRegistrator.register({
	url: 'http://localhost:3000',
	width: 1920,
	height: 1080,
});
```

Reply:
`tsx` and `global-jsdom` are intentional decisions.

Reply:
`global-jsdom` + `jsdom` is huge and slow. big -1 for them.
`tsx` is also a big dep because it's designed to be a full repl/ts execution env. And we only need transpilation, what `nodejs-loaders/tsx` do

Reply:
The statement "Huge and slow" is quite a bold one. Here's a breakdown:

- **global-jsdom** (including jsdom) takes up 13MB across 60 packages (59 if we exclude jsdom itself).
- **tsx** is 23MB across 5 packages.

That totals to around **36MB**.

On the other hand:

- **@happy-dom/global-registrator** is 14MB across 4 packages.
- **@nodejs-loaders/css-module** is 400KB across 6 packages.
- **@nodejs-loaders/alias** is 200KB across 2 packages.
- **@nodejs-loaders/tsx** is 23MB across 4 packages.

That sums up to approximately **37MB**. In the end, the sizes of these packages are quite comparable.